packages/utils/src/logger.ts

```diff
@@ -37,7 +37,8 @@ export const logStream = new StreamConcatenator()
 export const logger = pino(
   {
     level:
-      process.env.LOG_LEVEL || process.env.DEBUG === 'true' ? 'debug' : 'info',
+      process.env.LOG_LEVEL ||
+      (process.env.DEBUG === 'true' ? 'debug' : 'info'),
```

Comment:
Can we change this so to a simple truthy check for `DEBUG`? e.g. `process.env.DEBUG ? 'debug' : 'info'`

Reply:
https://github.com/zkopru-network/zkopru/commit/c012d5aaf995b3954e214bd957e92bea50ff39e2 resolves